perses/protocols/nonequilibrium_cycling.py

```diff
@@ -11,22 +11,39 @@
     ProtocolResult,
     ProtocolDAGResult,
     ProtocolUnitResult,
+    Context,
+    execute
 )
+
 #from gufe.protocols.settings import ...
 
 
-class InitializeUnit(ProtocolUnit):
-    def _execute(self, dependency_results):
+class GenerateHybridTopology(ProtocolUnit):
```

Comment:
Just a small naming note -- I think you will want this to be called something like `GenerateHybridTopologyFactory` as  the class that will need to be generated is called `HybridTopologyFactory`: https://github.com/choderalab/perses/blob/main/perses/annihilation/relative.py#L17

Reply:
And I think all of the variable names below that reference a "hybrid topology" should really be called "hybrid topology factory" or "htf"

Reply:
Thanks for the feedback, and yes, this are just place-holder names that were aimed at helping me get the structure of the `gufe` objects (protocols and units). The names themselves of these (namely the `GenerateHybridTopology`) won't end up being that, it was just to illustrate the example.

Reply:
Note that the `HybridTopologyFactory` generates more than just a hybrid topology -- it also (importantly) generates a hybrid system, which is why I think its important to use `HybridTopologyFactory` instead of `HybridTopology` in this PR

Reply:
Perhaps we could call this a `HybridAlchemicalFactory`, since it's supposed to take the real systems and generate a hybrid alchemical system? We could do that now or wait until we refactor, unify, and put this in `openmmtools`.

Reply:
This has been modified in the code. Resolving.